README.md

```diff
@@ -1,2 +1,44 @@
-# jumpstart-frontend
-The frontend for the C4C Jumpstart project
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
I don't think there's any reason to remove the initial readme. We'll just have to rewrite the script explanation.

Reply:
agreed